### PR TITLE
110244 install one plugin

### PR DESCRIPTION
**Pull request by ricardo-pcan**
# Related task
* [Poder instalar un solo plugin](http://manoderecha.net/md/index.php/task/110244)

# Issue description
We need that WW just can install a especific plugin

# Solution description

We modified fabric task `install_plugins` which does install all wordpress plugins (custom and thirth party). Until now this function does not receive arguments but in this change this function receive an argument which is the plugin name that requires install.
By the way, we create three functions which description is the next.
* `check_if_plugin_exist`: This function checks that the argument name be a plugin within setting.json
  :param name: This argument is the name of plugin that requires check
* `install_custom_plugin`: This function install a custom_plugin that is in the settings.json file
  :param custom_plugin: This is a dictionary with the custom_plugin data from setting.json
* `install_plugin`: This function install a plugin that is in the settings.json file
  :param plugin: This is a dictionary with the plugin data from setting.json

#### Install All Plugins
`fab environment:[environment] install_plugins`

seleccion_128

#### Install specific plugin
`fab environment:[environment] install_plugins:"all-in-one-seo-pack"`

seleccion_129

### CODE REVIEW

**Comment by haga87** [comment]
This won't update https://github.com/vinco/wordpress-workflow/blob/master/defaults/settings.json , will it?

So, it could leave a unstable installation on new vagrant provision, also any new plugin installed this way won't be reflected on git history.

**Comment by dgamboaestrada** [comment]
Héctor totally agree with you.

Ricardo, the idea of this funcinalidad, is to take settings from the json as the "install_plugins" task.

File: fabfile.py [hunk 1]
```diff
@@ -155,68 +155,135 @@ def activate_theme():
 
 
 @task
-def install_plugins():
+def install_plugins( name='' ):
     """
-    Installs plugins and initialize according to the settings.json file.
+    Installs plugins and initialize according to the settings.json file. :param name: This is an argument for install one specific plugin, if this is null install all plugins
```
Comment (dgamboaestrada) [inline fabfile.py]: Ricardo podrías arreglar esta linea, respetando el pep8, respecto a la longitud de la linea:

https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Comment (GloriaPG) [inline fabfile.py]: :P jefecito in action.

**Pull request merged**